configs/errors.eslintrc.json

```diff
@@ -26,6 +26,9 @@
     "camelcase": "off",
     "comma-dangle": "off",
     "curly": "error",
+    "object-curly-spacing": [ "error", "always" ],
+    "array-bracket-spacing": [ "error", "always" ],
+    "comma-spacing": "error",
```

Comment:
It's up to us to decide if we want spaces in arrays and objects, but at least it will be consistent everywhere and auto-saved in vscode.

Reply:
I don't know if this is the official description of the language, but on that site I see that they use spaces for curly brackets (objects) but not for square brackets (arrays):

https://www.typescriptlang.org/docs/handbook/basic-types.html#array